sdk/maps/Azure.Maps.Timezone/samples/MapsTimezoneSamples.md

```diff
@@ -0,0 +1,50 @@
+## Examples
+
+You can familiarize yourself with different APIs using our [samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Timezone/samples). 
```

Comment:
We don't need to add this sentence here because `/samples/README.md` also point to this page.
We also need to guide users what namespace they should import. Please refer to [Rendering sample](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/maps/Azure.Maps.Rendering/samples/MapsRenderingTilesImageriesImagesSamples.md)